op-node/rollup/derive/engine_queue.go

```diff
@@ -248,18 +241,16 @@ func (eq *EngineQueue) PendingSafeL2Head() eth.L2BlockRef {
 	return eq.ec.PendingSafeL2Head()
 }
 
-func (eq *EngineQueue) EngineSyncTarget() eth.L2BlockRef {
-	return eq.ec.EngineSyncTarget()
-}
-
 // Determine if the engine is syncing to the target block
 func (eq *EngineQueue) isEngineSyncing() bool {
 	return eq.ec.IsEngineSyncing()
 }
 
 func (eq *EngineQueue) Step(ctx context.Context) error {
-	if eq.needForkchoiceUpdate {
-		return eq.tryUpdateEngine(ctx)
+	// If we don't need to call FCU, keep going b/c this was a no-op. If we needed to
+	// perform a network call, then we should yield even if we did not encounter an error.
+	if err := eq.ec.TryUpdateEngine(ctx); err != errNoFCUNeeded {
```

Comment:
```suggestion
	if err := eq.ec.TryUpdateEngine(ctx); errors.Is(err, errNoFCUNeeded) {
```

I may have the args the wrong way around of `errors.Is` but it gives us the flexibility to wrap the returned error to provide more detail if needed in the future.

Reply:
good callout